internal/provider/provider.go

```diff
@@ -69,9 +87,10 @@ func (p *ConductoroneProvider) Configure(ctx context.Context, req provider.Confi
 	ServerURL := data.ServerURL.ValueString()
 	ClientID := data.ClientID.ValueString()
 	ClientSecret := data.ClientSecret.ValueString()
+	TenantDomain := data.TenantDomain.ValueString()
```

Comment:
how does this work if I only set ClientID and ClientSecret?

We should be infering the serverurl/tenant domain just based on the client id in that case